boards/Makefile.common

```diff
@@ -67,49 +64,13 @@ else
   endif
 endif
 
-
-# Check that rustc is the right nightly - warn not error if wrong, sometimes useful to test others
-RUSTC_VERSION_STRING := rustc 1.26.0-nightly (2789b067d 2018-03-06)
-ifneq ($(shell RUSTUP_TOOLCHAIN=$(RUSTUP_TOOLCHAIN) rustc --version), $(RUSTC_VERSION_STRING))
-  DUMMY := $(shell rustup install $(RUSTUP_TOOLCHAIN))
```

Comment:
This block is still necessary I think. It's responsible for ensuring the right version of rustc is installed